recipe/meta.yaml

```diff
@@ -24,13 +25,16 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6.2
-    - astroid >=2.9.0,<2.10
+    - python
+    - astroid >=2.11.6,<2.12.0
     - colorama  # [win]
+    - dill >=0.2
     - isort >=4.2.5,<6
     - mccabe >=0.6,<0.8
     - platformdirs >=2.2.0
-    - toml >=0.9.2
+    # 2022/07/25 RSK pylint switched from toml to tomli in 2.13.0
+    - tomli >=1.1.0
+    - tomlkit >=0.10.1
     - typing_extensions >=3.10
 
 test:
```

Comment:
Now we can enable pip check and drop the comment 
```
    # 2022/03/22: Disable pip check because
    # pylint 2.12.2 has requirement mccabe<0.7,>=0.6, but you have mccabe 0.7.0.
    # We can safely use mccabe 0.7.0 https://github.com/PyCQA/mccabe/compare/0.6.1...0.7.0, it was released after pylint 2.12.2 and it haven't broken changes
```
because now `mccabe` has constraints >`=0.6,<0.8`.

And try to add the command `pylint-config --help`